pkg/flare/archive.go

```diff
@@ -326,7 +326,32 @@ func zipExpVar(tempDir, hostname string) error {
 		}
 	}
 
-	return nil
+	apmPort := "8126"
+	if config.Datadog.IsSet("apm_config.receiver_port") {
+		apmPort = config.Datadog.GetString("apm_config.receiver_port")
+	}
+	resp, err := http.Get(fmt.Sprintf("http://127.0.0.1:%s/debug/vars", apmPort))
+	if err != nil || resp.StatusCode != http.StatusOK {
+		// trace-agent may not be running
```

Comment:
We should write that in the file (with the error code and the error ?). That way when we receive a flare we will know exactly why we were not able to get the info from the trace-agent.

Reply:
Good idea! Changed and wrote to file in both cases: err != nil and resp != 200